src/components/pages/career/career-page.tsx

```diff
@@ -11,13 +11,28 @@ import { useTranslation } from 'gatsby-plugin-react-i18next';
 import { SyPersonioJob } from '../../../types';
 import { ImageHero } from '../../content/heroes';
 import { IGatsbyImageData } from 'gatsby-plugin-image';
+import { ImageSpacer } from '../../pages/landingpage/image-spacer';
 
 interface CareerPageProps {
   positions: SyPersonioJob[];
   heroImageData: IGatsbyImageData;
+  officeImages: OfficeImages;
 }
 
-export const CareerPage = ({ positions, heroImageData }: CareerPageProps) => {
+interface OfficeImage {
+  relativePath: string;
+  childImageSharp: {
+    gatsbyImageData: IGatsbyImageData;
+  };
+}
```

Comment:
You created exactly the same interface in the parent. In my opinion it makes sense to delete this interface and import the one from the parent

Reply:
you are right, that makes more sense. Thanks for the suggestion , I have changed it.